app/server/api/v1/apisearch.js

```diff
@@ -55,21 +47,13 @@ router.get('/autocomplete/:term', jsonParser, (req, res, next) => {
       index: `contentful_mltlrs3kods6_en-us`,
       body: buildPrefixQuery(searchTerm)
     }).then(results => res.status(200).json(formatAutocompleteResults(results, searchTerm, true)))
-      .catch(err => {
-        /* eslint-disable */
-        console.error(err);
-        /* eslint-enable */
-        res.status(500).json({
-          'message': 'Internal error'
-        })
-      })
   } catch (err) {
     /* eslint-disable */
     console.error(err)
     /* eslint-enable */
-    res.status(500).json({
-      'message': 'Internal error'
-    })
+    let error = new Error()
```

Comment:
We need to get rid of this. Really we just want to pass on the `err` object so that we don't loose the error message. Does that make sense?